src/main/java/fr/esir/jxc/utils/CheckEmailFormat.java

```diff
@@ -0,0 +1,16 @@
+package fr.esir.jxc.utils;
+
+import java.util.regex.Pattern;
+
+public class CheckEmailFormat {
```

Comment:
```suggestion
public class EmailFormatChecker {
```

➜ c'est une classe pas une fonction, répercuter ça sur la fonction